### ISSUE TITLE
Accessibility Issue; Grey Button Contrast

### ISSUE THREAD

**Issue by magicislxnd** [Wilcolab/Anythink-Market-kalm96ag#2]
There is an issue with the color of the sign-in and sign-up buttons, as they are not easily visible due to their grey color. 

This should be modified as a certain level contrast is required to meet accessibility standards and provide services to a wider audience.
